jobbergate-api/jobbergate_api/apps/applications/routers.py

```diff
@@ -38,19 +45,27 @@ async def applications_create(
     """
     Create new applications using an authenticated user token.
     """
+    logger.debug(f"Creating {application=}")
+
     identity_claims = IdentityClaims.from_token_payload(token_payload)
     create_dict = dict(
         **application.dict(exclude_unset=True),
         application_owner_email=identity_claims.email,
     )
 
+    logger.debug("Inserting application")
+
     try:
         insert_query = applications_table.insert().returning(applications_table)
+        logger.debug(f"insert_query = {render_sql(insert_query)}")
         application_data = await database.fetch_one(query=insert_query, values=create_dict)
 
     except INTEGRITY_CHECK_EXCEPTIONS as e:
+        logger.warning(f"INTEGRITY_CHECK_EXCEPTIONS: {str(e)}")
```

Comment:
shouldn't it be logging.error?

Reply:
solved in [0b4f677](https://github.com/omnivector-solutions/jobbergate/pull/121/commits/0b4f677de19511164ead7888b0e8b72006ea43e1)